web/templates/settings/language_selection_widget.hbs

```diff
@@ -5,7 +5,7 @@
         {{> ../help_link_widget link=help_link_widget_link }}
         {{/if}}
     </label>
-    <button type="button" id="id_language_selection_button" class="language_selection_button button rounded tippy-zulip-delayed-tooltip" data-section="{{section_name}}" data-tippy-content="{{t 'Change language' }}">
+    <button type="button" id="id_language_selection_button" class="language_selection_button action-button action-button-quiet-neutral tippy-zulip-delayed-tooltip" data-section="{{section_name}}" data-tippy-content="{{t 'Change language' }}">
```

Comment:
Can this be updated to use `action_button.hbs` component? That would possibly need some changes to have `data-language-code` attribute on `button` element and also some JS changes. If it needs too much changes, then can be done in a separate commit.